header.php

```diff
@@ -177,11 +191,21 @@
 		<?php if ( is_archive() && $object->description ) : ?>
 			<p class="page-heroContent"><?php echo wp_kses_post( $object->description ); ?></p>
 		<?php endif; ?>
-		<?php if ( $hero_cta_text && ( $hero_cta_link || $hero_embed ) ) : ?>
+		<?php if ( $hero_cta_text && ( $hero_cta_link || $hero_embed ) || $hero_cta_two_text && $hero_cta_two_link ) : ?>
```

Comment:
If you're reliant on both the text and the link for the 2nd CTA, please enclose them in parentheses, so that it's more readily apparent